docs/finalproject/external-projects.md

```diff
@@ -16,6 +16,7 @@ Here are examples of web apps we built as a Final Project:
 - [Chatterbox Talents](https://ldn8-horizon-chatterbox-talent.netlify.app/)
 - [The Breteau Foundation](https://bugsquashers-edu.netlify.app/)
 - [Akwaaba](https://akwaaba.netlify.app/)
+- [Bike for Refugees](https://linktr.ee/callbackcats)
```

Comment:
@kfklein15 this is deployed on Heroku so is down. Everything our trainees built before November 2022 is down. If we put it up, we need to ask the trainees to redeploy on render.com.

Reply:
Which link is using Heroku? I opened all the links before adding them to the PR and they were all up and running.